database/rrddim.c

```diff
@@ -187,14 +187,14 @@ void rrdcalc_link_to_rrddim(RRDDIM *rd, RRDSET *st, RRDHOST *host) {
     for (rrdc = host->alarms_with_foreach; rrdc ; rrdc = rrdc->next) {
         if (simple_pattern_matches(rrdc->spdim, rd->id) || simple_pattern_matches(rrdc->spdim, rd->name)) {
             if (rrdc->hash_chart == st->hash_name || !strcmp(rrdc->chart, st->name) || !strcmp(rrdc->chart, st->id)) {
-                char *usename = alarm_name_with_dim(rrdc->name, strlen(rrdc->name), rd->name, strlen(rd->name));
-                if (usename) {
-                    if(rrdcalc_exists(host, st->name, usename, 0, 0)){
-                        freez(usename);
+                char *username = alarm_name_with_dim(rrdc->name, strlen(rrdc->name), rd->name, strlen(rd->name));
+                if (username) {
+                    if(rrdcalc_exists(host, st->name, username, 0, 0)){
+                        freez(username);
                         continue;
                     }
 
-                    RRDCALC *child = rrdcalc_create_from_rrdcalc(rrdc, host, usename, rd->name);
+                    RRDCALC *child = rrdcalc_create_from_rrdcalc(rrdc, host, username, rd->name);
```

Comment:
The name of the variable is not very good but it shouldn't be `username` anyway.

Reply:
Fwiw, I've run into `username` typo'd as [usename](https://www.postgresql.org/docs/9.3/view-pg-user.html) in the past. -- That wasn't fixable because it's an ancient published mistake that things rely on (as w/ http's [Referer](https://lists.w3.org/Archives/Public/ietf-http-wg-old/1995JanApr/0107.html)

Would `use_name` be ok? -- That seems to be the general convention here.

Reply:
It's a local variable with a very small scope. I think it can be simply `name`. @thiagoftsm did you have any special meaning in mind when you named the variable `usename`?

Reply:
You are right @vlvkobal  we can change for `name`, the initial intention was to say that the dimension needs `to use the name`.